### PR TITLE
add dependencies,because run the war in tomcat get the exception

### PR DESCRIPTION
**Pull request by hardzhang**
add dependencies,because run the war in tomcat get the exception follows:
Exception :java.lang.ClassNotFoundException:com.fasterxml.jackson.core.exc.InputCoercionException
because the class : com.fasterxml.jackson.core.exc.InputCoercionException only can found at least the dependency version is bigger than 2.10.1

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@hardzhang Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1877)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1877) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@hardzhang Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1877)!

File: samples/oauth2/sparklr/pom.xml [hunk 1]
```diff
@@ -157,6 +157,12 @@
 			<version>${jackson2.version}</version>
 		</dependency>
 
+   <dependency>
```
Comment (jgrandja) [inline samples/oauth2/sparklr/pom.xml:160]: The `POM` already includes:

```
		<dependency>
			<groupId>com.fasterxml.jackson.core</groupId>
			<artifactId>jackson-databind</artifactId>
			<version>${jackson2.version}</version>
		</dependency>
```

Which transitively imports `com.fasterxml.jackson.core:jackson-core`.

So I don't see why this change is needed?

**Comment by jgrandja** [comment]
@hardzhang I'm closing this PR due to lack of feedback from review.

If you still feel this is an issue, please submit a new PR and address the review feedback.
